addons/web/static/src/views/graph/graph_view.scss

```diff
@@ -1,83 +1,7 @@
-.o_graph_view > .o_content > .o_renderer {
-  height: 100%;
-
-  .o_graph_canvas_container {
-    padding-top: 5px;
-    position: relative;
-    height: 100%;
-    width: 100%;
-    top: 0px;
-    left: 0px;
+.o_graph_view > .o_content > .o_renderer .o_graph_canvas_container {
+    min-height: 250px;
+    
     canvas {
```

Comment:
There are canvas rendered in the graph view that shouldn't use `$o-view-background-color`?
If not (I guess so) simplify the selector: the resulting one is probably unnecessarily specific. 
```css
.o_graph_view > .o_content > .o_renderer .o_graph_canvas_container canvas {}
```